script/transpile-web-ui.py

```diff
@@ -68,10 +70,10 @@ def clean_target_dir(target_dir):
     except Exception as e:
         raise Exception("Error removing previous webpack target dir", e)
 
-
+#pylint: disable-msg=too-many-arguments
```

Comment:
we have limitation for maximum args in pylint, added this as quick fix cc @onyb @petemill

Reply:
I think we can also remove last `env` parameter as it is never passed to the function

Reply:
Either is fine by me. I wonder if this is better if we use js-style object for the webpack params. Or just have all the arg parsing happen in JS and not doubled up in python. Follow-ups of course.